### DISCUSSION TITLE
Deploy open data cam in azure

### DISCUSSION THREAD

**Discussion by onetwentytwo** [opendatacam/opendatacam#535] [Q&A] [answered]
Hello, 

Has anyone been successful with deploying to azure? Which linux distro did you use? Which vm did you use? 

Any pointers?

**Comment by vsaw** [comment]
I don't know anyone who tried, but it should work. 

You would need the same requirements as for the desktop environment (a Linux machine with CUDA installed).

**Comment by taz4535** [comment]
Have you gotten anywhere on this? We've just secured Azure funds so would be amazing if we can setup as a pilot.

**Comment by onetwentytwo** [comment, marked as answer]
It works, you need to get the VM that has a GPU. The only issue is it's a
very expensive VM.
